### PR TITLE
Add main component

### PR DESCRIPTION
**Pull request by Cortesn**

### CODE REVIEW

**Review by tdraxler — COMMENTED** [review]
Good stuff!

File: src/Components/main.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+class Main extends React.Component {
+    constructor(props) {
+        super(props);
+    }
+
+    render() {
+        return (
+            <div>
+                <h1>Welcome to Weather Roasted</h1>
```
Comment (tdraxler) [inline src/Components/main.js:11]: test comment. Please ignore

**Comment by mswarts** [comment]
looks good

**Review by mswarts — COMMENTED** [review]
submitting review...

File: src/Components/main.js (hunk 1 above)
Comment (Cortesn) [reply src/Components/main.js:11]: okay

**Review by tdraxler — APPROVED** [review]

**Review by mswarts — APPROVED** [review]

**Pull request merged**